### PR TITLE
Added roleactions mapping for cbo,musterroll,individuals and org..

### PR DESCRIPTION
**Pull request by sanjeet1137**

### CODE REVIEW

File: data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json [hunk 1]
```diff
@@ -85,6 +85,216 @@
...
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "MUSTER_ROLL_VERIFIER",
+      "actionid": 38,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "ORG_ADMIN",
```
Comment (jagankumar-egov) [inline data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json:132]: not required

File: data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json [hunk 2]
```diff
@@ -85,6 +85,216 @@
...
+      "actionid": 71,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "MUSTER_ROLL_VERIFIER",
```
Comment (jagankumar-egov) [inline data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json:180]: MUSTER_ROLL_VERIFIER require 37 action

File: data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json [hunk 3]
```diff
@@ -85,6 +85,216 @@
...
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "MUSTER_ROLL_APPROVER",
+      "actionid": 104,
+      "actioncode": "",
+      "tenantId": "statea"
+    },
+    {
+      "rolecode": "MUKTA_ADMIN",
```
Comment (jagankumar-egov) [inline data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json:210]: 62 & 62 actions need to be updated or removed in action-test

**Review by jagankumar-egov — CHANGES_REQUESTED** [review]
reviewed MUSTER_ROLL_VERIFIER & MUSTER_ROLL_Approver
can u check the action-test file it has a duplicate entry for 62 63

File: data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json (hunk 2 above)
Comment (sanjeet1137) [reply data/statea/ACCESSCONTROL-ROLEACTIONS/roleactions.json:180]: Line no 120 to 123 ,Having MUSTER_ROLL_VERIFIER for 37 action

**Comment by sanjeet1137** [comment]
Closing this PR as it was not synced with UAT branch

**Pull request closed without merging**